cpp/src/plasma/store.cc

```diff
@@ -73,19 +64,68 @@ using arrow::cuda::CudaDeviceManager;
 using arrow::util::ArrowLog;
 using arrow::util::ArrowLogLevel;
 
-namespace fb = plasma::flatbuf;
-
 namespace plasma {
 
+using flatbuf::MessageType;
+
 void SetMallocGranularity(int value);
 
 struct GetRequest {
-  GetRequest(Client* client, const std::vector<ObjectID>& object_ids);
+  GetRequest(asio::io_context& io_context,
```

Comment:
Although it's not the asio pattern, it may be necessary to pass these around by pointer to appease cpplint